assets/scss/components/_references.scss

```diff
@@ -0,0 +1,58 @@
+.references {
+    &__item {
+        display: block;
+        @include avoid-break;
+        &:not(:first-child) {
+            padding-top: 1.5rem;
+        }
+    }
+
+    &__header,
+    &__subheader {
+        display: block;
+        width: 100%;
+        height: 2.4rem;
```

Comment:
`__header` doesn't seem to be used, not sure if you intended for the reference name to have it!

Reply:
Also this could change to a `min-height` to handle possible weird long names